services/slackServices/shoutout.js

```diff
@@ -126,8 +153,83 @@ exports.submitDialog = async reqInfo => {
       title: 'Hurray, You are the newest shoutout Recipient',
       text: `You just received a shoutout from <@${reqInfo.userId}> on <#${org.channel_id}>`,
     };
-
     await slackModel.message.postOpenMessage(recipientAlert);
+
+    const dbInfo = {
+      giver_id: reqInfo.userId,
+      receiver_id: reqInfo.recipient,
+      message: reqInfo.content,
+      organization_id: org.slack_org_id,
+      access_token: org.access_token,
+    };
+
+    await ShoutOutHelper.saveToDatabase(dbInfo);
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+// for now let us only be able to return users recieved shoutouts
+exports.getUserShoutOuts = async reqInfo => {
+  try {
+    const org = await Organization.read(reqInfo.team);
+    const user = await User.readBySlackId(reqInfo.userId);
+    const id = user ? user.id : 0;
+    const userShoutouts = await ShoutOut.read(id);
+
+    // add an if conditionnto show when the don't have a shoutout
+    // post title message firsst
+    let message;
+    if (userShoutouts.length === 0 || !user) {
+      message = {
+        channel: reqInfo.channelId,
+        user: reqInfo.user_id,
+        token: org.access_token,
+        text:
+          "Yo! the selected user hasn't received or given a shoutout, you can start with `/bravo shoutout`",
+      };
+    } else {
+      message = {
+        channel: reqInfo.channelId,
+        user: reqInfo.user_id,
+        token: org.access_token,
+        text: `Here are the last ${userShoutouts.length} shoutouts sent and recieved by <@${reqInfo.userId}> 🎉\n`,
+      };
+    }
+
+    await slackModel.message.postMessage(message);
+
+    // loop through shoutouts and post each one
+    userShoutouts.map(async indiv => {
+      const giverSlackId = await User.readBySlackId(indiv.giverSlackId);
+      const receiverSlackId = await User.readBySlackId(indiv.receiverSlackId);
+      const date = indiv.created_at;
+
+      const messageList = {
+        channel: reqInfo.channelId,
+        user: reqInfo.user_id,
+        token: org.access_token,
+        attachments: JSON.stringify([
+          {
+            fallback: 'Message from Bravo',
+            callback_id: 'individual_shoutout',
+            attachment_type: 'default',
+            text: `\n <@${giverSlackId.slack_mem_id}> sent a shoutout to <@${receiverSlackId.slack_mem_id}> 🎉\n${indiv.message}`,
+            color: '#4265ED',
+            // please don't remove this code, very important
+            // actions: [
+            //   {
+            //     type: 'button',
+            //     text: 'View',
+            //     url: `${clientUrl}/shoutout/${indiv.id}`,
+            //   },
+            // ],
+            footer: `Bravo | ${moment(date).format('dddd, MMMM Do YYYY')}`,
+          },
+        ]),
+      };
+      await slackModel.message.postMessage(messageList);
+    });
   } catch (err) {
     console.log(err);
```

Comment:
You may have forgotten to remove the `console.log(err);`

Reply:
I have changed this to return null. It's not a part of any HTTP request so I'm not what to do with it as the other slack services follow the same pattern. Let me know how I should proceed.